app/models/partner.rb

```diff
@@ -74,9 +74,6 @@ class Partner < ApplicationRecord
   validates :twitter_handle,
             format: { with: TWITTER_REGEX, message: 'invalid account name' },
             allow_blank: true
-  validates :facebook_link,
```

Comment:
I don't think this is covered either -- this in relation to a Partner adding a link to their facebook page on their contact details, right? Not anything actually related to events (there are a few other places where it seems to refer to removing the contact details stuff rather than the events stuff too?)

Reply:
yeah this is to keep